src/Files.FullTrust/Helpers/DriveHelpers.cs

```diff
@@ -11,10 +11,10 @@ public static string GetVolumeId(string driveName)
             string name = driveName.ToUpperInvariant();
             string query = $"SELECT DeviceID FROM Win32_Volume WHERE DriveLetter = '{name}'";
 
-            using var searcher = new ManagementObjectSearcher(query);
-            foreach (ManagementObject item in searcher.Get()) // max 1 result because DriveLetter is unique.
+            using var cimSession = CimSession.Create(null);
```

Comment:
Changed to use MMI which we already use, instead of adding reference to System.Management.